src/util.h

```diff
@@ -1,30 +1,88 @@
 #include <fstream>
+#include <string.h>
+#include "solver.h"
 
-void file_to_grids(std::string filename, std::vector<Grid> &grids){
+
+struct StartupOptions
+{
+    std::string input_file = "simple-500";
+    Solver *solver = new BFS();
+    bool run_parallel = true;
+};
+
+StartupOptions parseOptions(int argc, const char **argv)
+{
+    StartupOptions rs;
+    for (int i = 1; i < argc; i++)
+    {
+        // For those with arguments
```

Comment:
I think something like this might be easier to maintain.
https://stackoverflow.com/questions/865668/parsing-command-line-arguments-in-c